fontforge/splineutil.c

```diff
@@ -1494,7 +1493,6 @@ static SplinePointList *SplinePointListSplitSpiros(SplineChar *sc,SplinePointLis
 	    memset(temp+(i-start),0,sizeof(spiro_cp));
 	    temp[i-start].ty = SPIRO_END;
 	    cur = SpiroCP2SplineSet( temp );
-	    free( temp ); temp = NULL;
```

Comment:
`SpiroCP2SplineSet` sets `cur->spiros=temp` (i.e. `temp` is used in the return value)